.changeset/three-emus-tell.md

```diff
@@ -0,0 +1,116 @@
+---
+"@salt-ds/lab": patch
+---
+
+- `DatePickerSinglePanel` has been renamed to `DatePickerSingleGridPanel`. This component now provides up to 12 `Calendars` within a panel via the `numberOfVisibleMonths` and `columns` props, enabling multi-month date selection (defaults to 1 visible calendar).
```

Comment:
The "renamed" here is misleading. `DatePickerSinglePanel` still existed with a special mapping to `DatePickerSingleGridPanel`. Maybe this should be 2 items, one being new one added, the other being "deprecated" to avoid immediate breakage of code

Reply:
good idea, done